modules/jaeger-rn-fixed-issues.adoc

```diff
@@ -15,6 +15,10 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
+* link:https://issues.redhat.com/browse/TRACING-2009[TRACING-2009] The Jaeger Operator has been updated to include support for Strimzi Kafka Operator 0.23.0.
+
+* link:https://issues.redhat.com/browse/TRACING-1907[TRACING-1907] The Jaeger agent sidecar injection was failing due to missing config maps in the application namespace. The config maps were getting automatically deleted due to an incorrect `OwnerReference` field setting and as a result, the application pods were not moving past the "ContainerCreating" stage. The incorrect settings have been removed.
```

Comment:
Wondering if ContainerCreating should be in backticks or bold? Not sure where (if anywhere) that text appears so don't know, just seems like a thing that might be in backticks or bold :)

Reply:
From the ticket I think it's a status message.  Do messages that appear in the console count as code? 🤔

Reply:
Typically, yes, messages in the GUI go in backticks. But if you're not 100% sure how/where this appears, I can't really see if being critical :)